src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -3,30 +3,59 @@
 $spacing: 15px;
 $close-icon-width: 20px;
 $message-margin: $spacing + $close-icon-width + 5px;
+$action-button-height: 30px;
+$action-button-padding: 6px;
 
-.announcement-container {
+.AnnouncementMessage {
   display: block;
-  z-index: z(announcement);
-  position: absolute;
+  position: relative;
   background: $primary;
-  top: 66px;
-  left: 50%;
-  width: 500px;
-  margin: 0 auto 0 -250px;
+  width: $announcement-container-width;
   transition: left 0.1s linear;
+  padding: $spacing 0;
+  border-radius: $border-radius--xl;
+  text-align: center;
 
-  &.centered {
-    left: 50%;
+  &--user {
+    left: 0;
+    right: 0;
+    margin: 0 auto;
+    position: absolute;
+    z-index: z(announcement-banner);
   }
 
-  padding: $spacing 0;
-  border-radius: 0 0 20px 20px;
-  text-align: center;
+  &--fullscreen {
+    top: 35%;
+    left: 35%;
+    position: fixed;
+    z-index: z(announcement-banner);
+    width: $announcement-container-width-fullscreen;
+  }
```

Comment:
can we make centering using flex?

![image](https://user-images.githubusercontent.com/63313732/133644203-2e0c3cea-d834-449e-a571-8f7071735400.png)

Reply:
fixed styles in ed799a1